mq/mq.go

```diff
@@ -67,11 +70,17 @@ func (q Queue) RunConsumer(consumer Consumer, cache storage.Addresses) {
 		return
 	}
 
+	if PrefetchCount < minPrefetchCount {
+		logger.Info("Change prefetch count to default")
+		PrefetchCount = defaultPrefetchCount
+	}
```

Comment:
How do you feel about `panic`ing here?

If not - consider to make it a `logger.warn` and to add the `PrefetchCount` and `minPrefetchCount` to the log statement so it's easier to fix

Reply:
I though about panic, but if we will have a broken container on production with auto restart - it is worse than having small optimized prefetch count

Reply:
Will change to warn